Experiments/Experiments/DefaultFeatureFlagService.swift

```diff
@@ -85,7 +85,7 @@ public struct DefaultFeatureFlagService: FeatureFlagService {
         case .customLoginUIForAccountCreation:
             return buildConfig == .localDeveloper || buildConfig == .alpha
         case .subscriptionProducts:
-            return buildConfig == .localDeveloper || buildConfig == .alpha
+            return true
```

Comment:
Leaving a note for future reference. We should be removing the feature flag totally, which I have taken care of in this PR https://github.com/woocommerce/woocommerce-ios/pull/11248